setup.py

```diff
@@ -28,9 +28,10 @@
 if 'linux' in _platform:
     extra_compile_args += ['-fopenmp']
     extra_link_args += ['-fopenmp']
-
+    include_dirs += ['/usr/include/eigen3', '/usr/include/libxml2']
```

Comment:
These are not useful hardcoded paths as they assume that eigen and libxml have been installed in those locations. However, if you are not using conda I could simply the specification of these paths by adding `setup.py` options for settings them? You could also just set `CFLAGS`.